packages/jest-snapshot/src/InlineSnapshots.ts

```diff
@@ -7,7 +7,12 @@
 
 import * as path from 'path';
 import type {PluginItem} from '@babel/core';
-import type {Expression, File, Program} from '@babel/types';
+import {
+  type Expression,
+  type File,
+  type Program,
```

Comment:
```suggestion
  Expression,
  File,
  Program,
```

just to match the way we do it in the rest of the source code